spec/requests/cloud_volume_types_spec.rb

```diff
@@ -0,0 +1,44 @@
+#
+# REST API Request Tests - Cloud Volume Types
+#
+# Regions primary collections:
+#   /api/cloud_volume_types
+#
+# Tests for:
+# GET /api/cloud_volume_types/:id
+#
+
+describe "Cloud Volume Types API" do
+  it "forbids access to cloud volume types without an appropriate role" do
+    api_basic_authorize
+    get(api_cloud_volume_types_url)
+    expect(response).to have_http_status(:forbidden)
+  end
+
+  it "allows access to cloud volume types with an appropriate role" do
+    api_basic_authorize action_identifier(:cloud_volume_types, :read, :collection_actions, :get)
+    instance = CloudVolumeType.create!
+    get(api_cloud_volume_types_url)
+    expect(response).to have_http_status(:ok)
+    url = api_cloud_volume_type_url(nil, instance)
+    expect(response.parsed_body["resources"]).to include("href" => url)
+  end
+
+  it "forbids access to a cloud volume type resource without an appropriate role" do
```

Comment:
Should these last two tests be wrapped in a context since they should both be testing the instance / resource?